src/test/java/org/folio/fqm/migration/strategies/MigrationDemonstration.java

```diff
@@ -0,0 +1,116 @@
+package org.folio.fqm.migration.strategies;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import org.folio.fql.service.FqlService;
+import org.folio.fqm.migration.MigratableQueryInformation;
+
+import java.util.List;
+import java.util.UUID;
+
+/**
+ * Demonstration class showing how the ContainsAnyToInOperatorMigration works
+ * This is a POC to show the migration approach for mapping old operators to new ones.
+ */
+public class MigrationDemonstration {
```

Comment:
I'm not sure what this class is here for. It's not a test and there is no reason for a `main`.